apps/er-coap/er-coap-observe.c

```diff
@@ -58,9 +58,9 @@ LIST(observers_list);
 /*---------------------------------------------------------------------------*/
 /*- Internal API ------------------------------------------------------------*/
 /*---------------------------------------------------------------------------*/
-coap_observer_t *
+static coap_observer_t *
```

Comment:
I have renamed the function as suggested. It would not be a problem to change the ```addr``` argument to be ```const``` but then the other functions in the CoAP API should also be updated in the same manner. I completely agree that this would be nice but is this something we should do in this PR or leave it for another PR?